xml/admin_cephx.xml

```diff
@@ -843,4 +843,73 @@ exported keyring for client.admin
    </variablelist>
   </sect2>
  </sect1>
+ <sect1 xml:id="permissions-cephx">
+   <title>Authorization Permissions (capabilities) for &cephx;</title>
+   <para>
+     &ceph; uses the term <literal>capabilities</literal> (caps) to describe
```

Comment:
I did not understand what "(caps)" meant. I was wondering if it was IN CAPITALS or something. I suggest spelling it out: "`capabilities`, or ''caps'' for short".